src/components/common/MwButton/MwButton.tsx

```diff
@@ -16,7 +14,7 @@ const MwButton = (props: MwButtonProps): JSX.Element => {
   };
 
   return (
-    <button type={type} className={className} onClick={handleChange}>
+    <button type={type} className={`mwButton ${className}`} onClick={handleChange}>
```

Comment:
```suggestion
    <button type={type} className={`mwButton${className ? className : ''}`} onClick={handleChange}>
```
Otherwise not passing a `className` to a component will result in a class name that looks like that: `mwButton undefined`

Reply:
Thank you, I missed that